src/scripts/destinyTrackerApi/bulkFetcher.js

```diff
@@ -49,11 +49,20 @@ class BulkFetcher {
    * @memberof BulkFetcher
    */
   bulkFetch(storesContainer) {
-    const stores = storesContainer.stores;
+    if (storesContainer.stores) {
+      const stores = storesContainer.stores;
 
-    this._getBulkFetchPromise(stores)
-      .then((bulkRankings) => this.attachRankings(bulkRankings,
-                                                  stores));
+      this._getBulkFetchPromise(stores)
+        .then((bulkRankings) => this.attachRankings(bulkRankings,
+                                                    stores));
+    }
+    else if (storesContainer.vendors) {
+      const vendors = $.map(storesContainer.vendors, (vendor) => { return vendor; });
```

Comment:
I thought so too, but an object.forEach that gets called later on was throwing fits until I coerced it to an array.